src/mmw/js/src/modeling/gwlfe/entry/templates/field.html

```diff
@@ -14,6 +8,7 @@
             {% if maxValue != null %} max="{{ maxValue }}" {% endif %}
             placeholder="{{ autoValue | round(3) }}"
             value="{{ userValue }}"
+            {% if readOnly %} disabled {% endif %}
```

Comment:
Should these disabled inputs have a title/popover that indicates that they're readonly and why?

Reply:
They are quite evidently not editable, not sure if a popup indicating they are "read only" would add anything. As to why, that's due to them being "virtual BMPs" and not true ones, i.e. those that differ from a true BMP only in terms of efficiency, so with the efficiency changed they mean nothing.

This is a pretty deep part of the app where we expect only expert users. I don't think we need to add any explanation here, but I'll re-evaluate this if the clients ask for that explicitly.